MakerTracker/Controllers/DashboardController.cs

```diff
@@ -11,7 +9,7 @@
 namespace MakerTracker.Controllers
 {
     [Authorize()]
-    public class DashboardController : Controller
+    public class DashboardController : BaseController
```

Comment:
I don't see the file that contains BaseController

Reply:
It's already there. It's not in the PR because I didn't create/edit it. It's in use on several other controllers, I'm just adding it to the files that didn't use it. It loads the profile picture and the user's display name into the view.